clang/test/Driver/sycl-offload.c

```diff
@@ -87,6 +87,13 @@
 
 /// ###########################################################################
 
+/// Validate SYCL option values
+// RUN:   %clang -### -fsycl-device-code-split=bad_value -fsycl  %s 2>&1 \
+// RUN:   | FileCheck -check-prefix=CHK-SYCL-BAD-OPT-VALUE %s
+// RUN:   %clang -### -fsycl-link=bad_value -fsycl  %s 2>&1 \
+// RUN:   | FileCheck -check-prefix=CHK-SYCL-BAD-OPT-VALUE %s
+// CHK-SYCL-BAD-OPT-VALUE: error: invalid value 'bad_value' in '{{.*}}'
```

Comment:
I'd like to see a true check for the option name instead of a wildcard, maybe as a `-Doption` value to FileCheck.

Reply:
Done.